cypress/integration/regression/decision-tree-regression-pt2.spec.ts

```diff
@@ -0,0 +1,255 @@
+  })
+  it('should collapse charts | step 20', () => {
+    decisionTreesPage.visit(`/filter?ds=${datasetName}`)
+    decisionTreesPage.decisionTreeResults.addAttribute.click()
+    decisionTreesPage.attributesList.searchForAttr.eq(0).type('aller')
+    decisionTreesPage.decisionTreeResults.graphHeaders.eq(0).click()
+    decisionTreesPage.attributesList.selectAll.contains('Select All').click()
+    cy.intercept('POST', '/app/statunits').as('applyAttributes')
+    decisionTreesPage.attributesList.addSelectedAttributes.click()
+    cy.wait('@applyAttributes').its('response.statusCode').should('eq', 200)
+    decisionTreesPage.decisionTreeResults.excludeInfo
+      .first()
+      .should('have.text', includedVariants)
+    decisionTreesPage.decisionTreeResults.collapseAll.first().click()
+    decisionTreesPage.decisionTreeChart.dataCharts.element.should('not.exist')
+  })
+  it('should expand collapsed charts | step 21', () => {
+    decisionTreesPage.visit(`/filter?ds=${datasetName}`)
+    decisionTreesPage.decisionTreeResults.addAttribute.click()
+    decisionTreesPage.attributesList.searchForAttr.eq(0).type('aller')
+    decisionTreesPage.decisionTreeResults.graphHeaders.eq(0).click()
+    decisionTreesPage.attributesList.selectAll.contains('Select All').click()
+    cy.intercept('POST', '/app/statunits').as('applyAttributes')
+    decisionTreesPage.attributesList.addSelectedAttributes.click()
+    cy.wait('@applyAttributes').its('response.statusCode').should('eq', 200)
+    decisionTreesPage.decisionTreeResults.excludeInfo
+      .first()
+      .should('have.text', includedVariants)
+    decisionTreesPage.decisionTreeResults.collapseAll.first().click()
+    decisionTreesPage.decisionTreeChart.dataCharts.element.should('not.exist')
+    decisionTreesPage.decisionTreeResults.expandAll.first().click()
+    decisionTreesPage.decisionTreeChart.dataCharts.element.should('exist')
+  })
+
+  it('should open text editor | step 22', () => {
+    decisionTreesPage.visit(`/filter?ds=${datasetName}`)
+    decisionTreesPage.decisionTreeResults.addAttribute.click()
+    decisionTreesPage.attributesList.searchForAttr.eq(0).type('aller')
+    decisionTreesPage.decisionTreeResults.graphHeaders.eq(0).click()
+    decisionTreesPage.attributesList.selectAll.contains('Select All').click()
+    cy.intercept('POST', '/app/statunits').as('applyAttributes')
+    decisionTreesPage.attributesList.addSelectedAttributes.click()
+    cy.wait('@applyAttributes').its('response.statusCode').should('eq', 200)
+    decisionTreesPage.decisionTreeResults.excludeInfo
+      .first()
+      .should('have.text', includedVariants)
+    decisionTreesPage.decisionTreeMenu.textEditor.click()
+    decisionTreesPage.decisionTreeResults.modalHeader.element.should(
+      'be.visible',
+    )
+  })
+
+  // eslint-disable-next-line jest/no-disabled-tests
+  it.skip('should save dataset and open it in main table | step 23', () => {
```

Comment:
Why it's disabled?

Reply:
there is a bug in an interface or a plug instead of functionality which will be implemented in nearest future)